surefire-its/pom.xml

```diff
@@ -241,6 +242,13 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>apache.ci</id>
+      <properties>
+        <its.forkCount>0.4C</its.forkCount>
+        <its.threadCount>1</its.threadCount>
+      </properties>
+    </profile>
```

Comment:
maybe we can detect CI by some of commons environment variable like CI ...?

do we need this only on Jenkins?

Reply:
it looks to GH nodes are not shared and so support more load.